silhouette/Graphtec.py

```diff
@@ -409,7 +409,7 @@ def read(s, size=64, timeout=5000):
     elif isinstance(data, bytearray):
         return str(data).decode()
     else:
-        return data.tostring().decode()
+        return data.tobytes().decode()  # return data.tostring().decode()
```

Comment:
not compatible with python2:
AttributeError: 'array.array' object has no attribute 'tobytes'

thus another solution is needed to keep compatiblity with python2

Reply:
resolved in master:
`try: ...tobytes()... exept: ...tostring()...`